src/odemis/gui/util/img.py

```diff
@@ -1226,8 +1228,8 @@ def spectrum_to_export_data(proj, raw, vp=None):
         text_colour = (0, 0, 0)
 
         # Draw bottom horizontal scale legend
-        tick_spacing = client_size[0] // 4
-        font_size = client_size[0] * SPEC_FONT_SIZE
+        tick_spacing = client_size[0] // 6
+        font_size = client_size[0] * 0.035
```

Comment:
Please use a constant. Actually, I'd recommend updating this constant to 0.035. And if it's too big for the other usage (like in AR display, a different constant should be used there, eg AR_FONT_SIZE)